src/DynamoCore/Library/TypedParameter.cs

```diff
@@ -13,7 +13,8 @@ public class TypedParameter
         private string summary = null; // Indicating that it is not initialized.
         private readonly string defaultValueString;
 
-
+        public bool nameIsValid { get; set; } = true;
```

Comment:
a public property needs a /// summary and should be named PascalCase 
https://en.wikipedia.org/wiki/Naming_convention_(programming)

Reply:
this is going to be serialized because it's not ignored... I don't think we can do that easily at this point - it's a schema change from the cogs schema. This bool should be ignored if possible from serialization - please check if TypedParameter has a converter for save/load - I think it may , in which case you're done! - if it uses attributes then you may need to ignore this property with [JSONIGNORE]

Reply:
indeed there is a typedParameter converter for serialization:
https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Graph/Workspaces/SerializationConverters.cs#L900

im not sure how deserialization works though - would be worth checking into I think.